home/ivp-planning/advanced-tools/local-library.md

```diff
@@ -0,0 +1,56 @@
+# Local Library
+
+You can import your own files/objects from your PC into iVP by using the "Import" option, found under the "File" menu at the top of the application.  
+
+![](../../../.gitbook/assets/3d-import-file.png)
+ 
+
+When importing your file, you have the option to change its parameters, as well as the option to add it to your library, which then will show up at the bottom of the default library on the left side of the application.
+
+![](../../../.gitbook/assets/3d-import-addtolibrary.png)
+
+
+### iVP Library  
+
+Once you have added an item to your library, the folder "iVP Library" will appear at the bottom of the library panel.
+Every item that you import, and chose to add to your library, can be found inside the "iVP Library" folder.  
+
+
+![](../../../.gitbook/assets/3d-import-library.png)
+
+### Editing your iVP Library
+
+When hovering an entry in the library, a button with a folder icon will appear. Clicking it will open the respective folder or select the respective file in your file browser  
+<img src="../../../.gitbook/assets/3d-import-ivplibrary.png" alt="" data-size="line">
+  
+Your "iVP Library" folder will be created in the default iVP-Directory.  
+You can click on the folder button (highlighted below) next to any object inside the folder to open the directory.  
+  
+The folders name can be changed at any time by editing the folder directly in the directory, which  
```

Comment:
Half of the sentence is missing and also I have no idea what you mean by "the folder name can be changed".